tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -252,5 +252,63 @@ public void BuildsApplication_IgnoresRuntimeVersionBasedSdkVersion_AndUsesSdkVer
                 },
                 result.GetDebugInfo());
         }
+
+        [Fact]
+        public void BuildsDotNetCore50App_UsingSdkFromGlobalJson()
```

Comment:
Will it help if we don't name it specifically as dotnetcore50app but instead add it as InlineData for future use?

Reply:
Sure makes sense. Will update it.

Reply:
I added this as part of an existing test's inline data but changed this particular test's name to indicate that we are `testing a preview version from global.json sdk version`